src/webview/components/memory-table.tsx

```diff
@@ -94,10 +93,10 @@ export const MoreMemorySelect: React.FC<MoreMemorySelectProps> = ({ count, offse
 };
 
 interface MemoryTableProps extends TableRenderOptions, MemoryDisplayConfiguration {
+    userReadArguments: Required<DebugProtocol.ReadMemoryArguments>;
```

Comment:
I think different names for `userReadArguments` and `memoryReadArguments` could help to indicate their purpose. If I understand correctly, `userReadArguments` are the initial read arguments, and `memoryReadArguments` are the current read arguments after user interaction/scrolling.

Reply:
Thinking about it again, `userReadArguments` is actually good. May just need to reconsider `memoryReadArguments`